src/components/Spinner.jsx

```diff
@@ -1,7 +1,18 @@
 import React from 'react';
+import { Audio } from 'react-loader-spinner';
 
 const Spinner = () => (
-  <div className="spinner">Loading...</div>
+  <div className="spinner">
+    <Audio
+      height="80"
+      width="80"
+      radius="9"
+      color="green"
+      ariaLabel="loading"
+      wrapperStyle
+      wrapperClass
```

Comment:
- Please, No inline styling.  inline styles are at the lowest level of the cascade as they’ll override all other styles, so you need to be careful when using it.